lmcache/storage_backend/local_backend.py

```diff
@@ -48,7 +48,7 @@ def __init__(
         self.update_lock = threading.Lock()
         
         # FIXME(Jiayi): `use_pin_memory` and `dst_device` should be configged dynamically
-        self.use_pin_memory = True
+        self.use_pin_memory = False
```

Comment:
Why `False` here

Reply:
Did some local tests and found that `pin_memory = True` will lead to some extra overhead. Maybe there are some performance bugs. 

Created another ticket #107 for this. Let's investigate that later